fendermint/vm/interpreter/src/fvm/state/genesis.rs

```diff
@@ -177,6 +177,17 @@ where
         }
     }
 
+    /// Flush the data to the block store. Returns the state root cid and the underlying state store.
+    pub fn finalize(self) -> anyhow::Result<(Cid, DB)> {
```

Comment:
It might be worth to either explain the difference between this and finalise in comment or remove one a keep the other? Now it's a bit ambiguous.

Reply:
I will remove `commit` method once this PR is merged, currently the old implementation is still there.